rllib/agents/alpha_star/alpha_star.py

```diff
@@ -58,6 +61,12 @@
         # old (replayed) ones.
         "replay_buffer_replay_ratio": 0.5,
 
+        # Tuning max_requests_in_flight_per_sampler_worker and
+        # max_requests_in_flight_per_learner_worker is important so backpressure is
+        # created on the remote workers and the object store doesn't fill up
+        # unexpectedly. If the workers spend time idle, consider increasing these.
+        "max_requests_in_flight_per_sampler_worker": 2,
+        "max_requests_in_flight_per_learner_worker": 2,
```

Comment:
is it common to use the same max_requests_in_flight for both sampler and learner workers?
for learner worker to keep up with the sampler worker, shouldn't it have more outstanding requests?

Reply:
Good question. But this would need to be tuned, I guess.